build/build-buildimages.sh

```diff
@@ -81,7 +81,10 @@ touch $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 > $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 
 # Build buildpack images
-source $REPO_DIR/build/build-buildpacks-images.sh
+# 'pack' is not supported on Windows
+if [[ "$OSTYPE" == "linux-gnu" ]] || [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
@dorfire I ran into this while trying to build the build image on my windows machine

Reply:
the only problem is with `pack create-builder` - it isn't implemented on Windows.
did you get an error from a different part?

Reply:
Yeah, the error I got was `ERROR: `pack create-builder` is not implemented on Windows.`

Reply:
well then, if we add an OS gate to `build-buildimages.sh` -
shouldn't we remove the existing OS gate from `install-pack.sh`?
And only leave the Could not detect compatible pack binary platform condition?

Reply:
But you can still run `build-buildpacks-images.sh` by itself though right, in which case you might still want to keep the OS gate in the other file

Reply:
`install-pack.sh` doesn't need to check if it's running on Windows. It only needs to check if it's on Linux or Mac, so it can download the correct binary.
Thus, I'm suggesting to remove the special Windows treatment for it, and leave it only in `build-buildpacks-images.sh`

Reply:
Updated.

Reply:
I have reverted my changes in all other files to keep it simple since I found that even though I need to have checks in OS type in `build-buildpacks-images` for quick exit, I also needed the same checks again in `install-pack.sh` (because I cannot use the OSTYPE to get the tar file i.e using `linux-gnu` vs `linux` when getting the tar file).